robot/Cumulus/resources/NPSP.py

```diff
@@ -1669,4 +1669,22 @@ def query_and_store_records_to_delete(self,object_name,**kwargs):
             for i in records:
                 self.salesforce.store_session_record(object_name,i)
 
+    @capture_screenshot_on_error
+    def verify_table_contains_row(self,table_name,record,**kwargs):
```

Comment:
Is this keyword generic enough for any usecase where we wanted to validate table contains row ?

Reply:
it works as long as it is a lightning table and has qalocator. I am using this in the Gift Entry and NPSP settings page. Since we are moving towards adding identifiers, I have added this here